app/models/room.rb

```diff
@@ -3,6 +3,7 @@
 class Room < ApplicationRecord
   belongs_to :user
   has_many :recordings, dependent: :destroy
+  has_many :room_meeting_options, dependent: :destroy
```

Comment:
I think we should add this `has_many :meeting_options, through: :room_meeting_options`

Reply:
I think this not still needed?